virttest/libvirt_xml/devices/filesystem.py

```diff
@@ -10,7 +10,7 @@
 
 class Filesystem(base.TypedDeviceBase):
 
-    __slots__ = ('accessmode', 'source', 'target')
+    __slots__ = ('accessmode', 'source', 'target', 'driver', 'binary', 'address', 'alias')
```

Comment:
I don't find any implementation for address attribute

Reply:
Updated.